erlang22/README.md

```diff
@@ -0,0 +1,20 @@
+# erlang22
+
+A programming language for massively scalable soft real-time systems.
+
+## Maintainers
+
+* The Habitat Maintainers: <humans@habitat.sh>
+
+## Type of Package
+
+Binary package
+
+## Usage
+
+Simple example of invoking `erl`:
+
+```
+hab pkg install core/erlang20
+hab pkg exec core/erlang20 erl
```

Comment:
should this be core/erlang22?

Reply:
Yup, missed the readme earlier. Fixed now